CMSgov/mint-app
# [EASI-4502] MDM-NCBP Solution

# EASI-4502

## Description


- Added migration for `MDM_NCBP` - contacts, need link, enum
- Extended GQL solution keys for `MDM_NCBP`
- Added translations
- Updated unit tests


## How to test this change


- Navigate to help center and verify help article matches Figma
- To verify need link, select `Non-Claims-Based Payments` on page one of payments
- Got to solution tracker and verify `MDM-NCBP` appears as a default for that need

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
